x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_creation_ui/pages/rule_editing/index.tsx

```diff
@@ -207,13 +210,21 @@ const EditRulePageComponent: FC<{ rule: RuleResponse }> = ({ rule }) => {
     dataViewId: defineStepData.dataViewId,
   });
 
+  const isRuleCustomizationDisabled = !isRulesCustomizationEnabled && rule?.immutable;
```

Comment:
This logic is also repeated in the above changes where we define the `activeStep` state. Can we re-use this const above?

Reply:
Sure, I reused the constant where applicable